pkg/R/sparkRBackend.R

```diff
@@ -0,0 +1,93 @@
+# Methods to call into SparkRBackend. 
+
+
+# Returns TRUE if object is an instance of given class
+isInstanceOf <- function(jobj, className) {
+  stopifnot(class(jobj) == "jobj")
+  cls <- SparkR:::callJStatic("java.lang.Class", "forName", className)
+  SparkR:::callJMethod(cls, "isInstance", jobj)
+}
+
+# Call a Java method named methodName on the object
+# specified by objId. objId should be a "jobj" returned
+# from the SparkRBackend.
+callJMethod <- function(objId, methodName, ...) {
+  stopifnot(class(objId) == "jobj")
```

Comment:
Hmm - So I am not really sure having this assert makes a big performance difference. On the other hand it might be useful to debug cases where we pass in something that is not a jobj. I'd say lets keep it till we do some perf testing ?